rules/linux/process_creation/proc_creation_lnx_device_driver_discovery.yml

```diff
@@ -0,0 +1,26 @@
+title: Detect Device Driver Discovery - Linux
+id: 23654e63-1b3d-4330-8303-fb680a3a4ab8
+status: test
+description: Adversaries may attempt to enumerate local device drivers on a victim host.This rule detects the use of commands such as 'lsmod' and 'modinfo' for device driver discovery on Linux systems.
+references:
+    - https://attack.mitre.org/techniques/T1652/
```

Comment:
As explained in the other PRs, please dont use this as a ref and give me more references to the tools